seed/data/dev/01500_court_reports.js

```diff
@@ -18,7 +18,7 @@ exports.seed = function (knex, Promise) {
       var workloadsToInsert = []
       for (var workloadOwner in workloadOwners) {
         for (var i = 0; i < 5; i++) {
-          workloadsToInsert.push(Object.assign({}, workloadRow, {workload_owner_id: workloadOwners[workloadOwner].id, staging_id: i + 1}))
+          workloadsToInsert.push(Object.assign({}, workloadRow, {workload_owner_id: workloadOwners[workloadOwner].id, staging_id: i + 1, workload_report_id: i + 2}))
```

Comment:
Does `workload_report_id` need to be an actual existing workload report id, so the joins in the views will work?

Reply:
No the views join to the workload_report through the court_reports_calculation table. So not necessary to be exact.  But changed anyway, now add the same set of court-reports for each existing workload_report